acmeproxy/proxy/tests/test_views.py

```diff
@@ -0,0 +1,324 @@
+import pytest
+
+from acmeproxy.proxy.models import Authorisation, Response
+
+
+@pytest.mark.django_db
+class TestAuthroisation:
+    def test_create_authorisation(self, client):
+        resp = client.post("/create_authorisation", data={"name": "example.com"})
+        assert resp.status_code == 200
+        assert resp.json()["result"]["authorisation"] == "example.com"
+
+    def test_create_authorisation_mixed_case(self, client):
+        resp = client.post("/create_authorisation", data={"name": "exAMplE.cOm"})
+        assert resp.status_code == 200
+        assert resp.json()["result"]["authorisation"] == "example.com"
+
+    def test_create_authorisation_missing_name(self, client):
+        resp = client.post("/create_authorisation", data={})
+        assert resp.status_code == 400
+
+    def test_create_authorisation_set_secret(self, client, settings):
+        settings.ACMEPROXY_AUTHORISATION_CREATION_SECRETS = {
+            "18084e750a1cff6f2d627e7a568ab81a": {"name": "developers"}
+        }
+        resp = client.post(
+            "/create_authorisation",
+            data={"name": "example.com", "secret": "18084e750a1cff6f2d627e7a568ab81a"},
+        )
+        assert resp.status_code == 200
+        assert resp.json()["result"]["authorisation"] == "example.com"
+
+    def test_create_authorisation_wrong_secret(self, client, settings):
+        settings.ACMEPROXY_AUTHORISATION_CREATION_SECRETS = {
+            "18084e750a1cff6f2d627e7a568ab81a": {"name": "developers"}
+        }
+        resp = client.post(
+            "/create_authorisation",
+            data={"name": "example.com", "secret": "wrong_secret"},
+        )
+        assert resp.status_code == 403
+        assert resp.json()["result"] is False
```

Comment:
It seems like these tests are covered by the parametrised tests that follow them.

Do we still need them?

```suggestion
```